src/operator/timeout.ts

```diff
@@ -97,6 +93,6 @@ class TimeoutSubscriber<T> extends Subscriber<T> {
   }
 
   notifyTimeout(): void {
-    this.error(this.errorToSend);
+    this.error(new TimeoutError());
```

Comment:
I'd propose to create error object in ctor, allows bit easier stack to trace (was done in https://github.com/ReactiveX/rxjs/pull/2132)